.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,60 @@
+name: Feature request
+description: Suggest an idea for Cocos
+labels: ["Needs Triage","Feature Request"]
+body:
+
+- type: markdown
+  attributes:
+    value: |
+      - Please write a descriptive proposal title above.
+      - Search [open](https://github.com/cocos/cocos-engine/issues) and [closed](https://github.com/cocos/cocos-engine/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been suggested.
+
+- type: textarea
+  attributes:
+    label: Use Case
+    description: Explain your use case, context, and rationale behind this feature request. More importantly, what is the project you are trying to build that led to the need for this feature? This is important to know the **context** in which the feature is being proposed.
+    placeholder: Example - "I am working on a MOBA game that ... With heavy use of collision detection..."
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Problem Description
+    description: Please add a clear and concise description of the problem or limitation you are having in your project.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Proposed Solution
+    description: Describe the solution you'd like in a clear and concise manner.
```

Comment:
Propose the desired workflow or API set to satisfy the need in a clear and concise manner.